scio-parquet/src/main/java/com/spotify/scio/parquet/avro/ParquetAvroSink.java

```diff
@@ -106,11 +107,14 @@ public ParquetAvroWriter(
     @Override
     protected void prepareWrite(WritableByteChannel channel) throws Exception {
       BeamOutputFile outputFile = BeamOutputFile.of(channel);
+      // https://github.com/apache/parquet-mr/tree/master/parquet-hadoop#class-parquetoutputformat
+      int rowGroupSize = conf.get().getInt(ParquetOutputFormat.BLOCK_SIZE, 128 * 1024 * 1024);
```

Comment:
@nevillelyh Just noticed that we have a `ParquetWriter.DEFAULT_BLOCK_SIZE` , same value, but probably better instead of keeping track of this ourselves?

Reply:
Yeah I'll use that. Parquet's constants are all over the place 🤷